maxtext_gpu_dependencies.Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG BASEIMAGE=maxtext_base_image
+FROM $BASEIMAGE
+
+
+RUN sed -i -e "s/main/main contrib/" /etc/apt/sources.list.d/debian.sources
+RUN curl https://developer.download.nvidia.com/compute/cuda/repos/debian12/x86_64/cuda-keyring_1.1-1_all.deb -o cuda-keyring_1.1-1_all.deb
```

Comment:
Can we ask our CUDA friends how we keep this up to date for whatever is recommended for Jax? (Like do they update the .deb or do they author new ones? Is there one that is always "last stable"?)